packages/api/src/controllers/access-control.ts

```diff
@@ -150,6 +151,16 @@ app.post(
         res.status(204);
         return res.end();
       case "jwt":
+        if (req.body.jwt) {
```

Comment:
If the type is jwt, isn't this payload field obligatory? Looking like a redundant check, or maybe we want an inverted check right in the beginning to return an error/403 right away if a JWT is not provided